data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -342,13 +342,26 @@
             [/then]
         [/if]
 
-        {CLEAR_VARIABLE random,sneak_up}
+        {CLEAR_VARIABLE random}
     [/event]
 
     # starting dialogue
     [event]
         name=start
 
+        [if]
+            [variable]
+                name=sneak_up
+                boolean_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Nym
+                    message= _ "Thanks to that map we got from the orcs last night, our scouts have been able to confirm the layout of the surrounding terrain, including the location of an orcish camp up ahead! I wonder if we could sneak around without alerting them. Oh my, what’s that far off in the distance? Are those..."
```

Comment:
The bit from "Oh my" onwards doesn't feel like it flows into the next `[message]` to me. I'd expect it to be something about enemy units that she's seen, but in the next line she switches to talking about mountains. If big mountains are within scouting range then she should have already seen them before the scouts were sent.